libs/wingc/src/parser.rs

```diff
@@ -885,10 +885,55 @@ impl<'s> Parser<'s> {
 		}
 
 		if module_name.name.starts_with("\"") && module_name.name.ends_with("\"") {
+			// we need to inspect the npm dependency to figure out if it's a JSII library or a Wing library
+			// first, find where the package.json is located
+			let module_name_parsed = module_name.name[1..module_name.name.len() - 1].to_string();
+			let source_dir = Utf8Path::new(&self.source_name).parent().unwrap();
+			let module_dir = wingii::util::package_json::find_dependency_directory(&module_name_parsed, &source_dir)
+				.ok_or_else(|| {
+					self
+						.with_error::<Node>(
+							format!(
+								"Unable to load \"{}\": Module not found in \"{}\"",
+								module_name, self.source_name
+							),
+							&statement_node,
+						)
+						.err();
+				})?;
+
+			// If the package.json has a `wing` field, then we treat it as a Wing library
```

Comment:
Note. When we were modifying the user's `package.json` in-place (in a previous implementation), users wouldn't really be required to run `wing pack` to test a new version of the library locally -- they could just have package.json point to their local Wing library directory.

But since we're no longer modifying `package.json`, the package.json may be missing the "wing" flag or its "main" field may be invalid, and that strategy doesn't work anymore. I've cheated in our monorepo by adding `"wing": true` to the package.json and adding a stub `index.js` file.

Reply:
Wow great point. I think we definitely want to support monorepos. Let's open an issue and discuss there.